src/arcade_collection/output/parse_growth_file.py

```diff
@@ -0,0 +1,143 @@
+import json
+from os import path
+import tarfile
+from typing import List, Union
+
+import ntpath
+import numpy as np
+import pandas as pd
+from prefect import task
+
+
+GROWTH_COLUMNS = [
+    "TICK",
+    "SEED",
+    "U",
+    "V",
+    "W",
+    "Z",
+    "POSITION",
+    "POPULATION",
+    "STATE",
+    "VOLUME",
+    "CYCLE",
+]
+
+
+@task
+def parse_growth_file(tar: tarfile.TarFile) -> pd.DataFrame:
+    """
+    Parse the tumor growth tar file.
+
+    Parameters
+    ----------
+    tar :
+        Tar file of simulations.
+
+    Returns
+    -------
+    :
+        Data of all timepoints of all simulations in tar file.
+    """
+    all_timepoints = []
+    for member in tar.getmembers():
+        extracted_member = tar.extractfile(member)
+        if extracted_member is not None:
+            base = ntpath.basename(member.name)
+            base_file = path.splitext(base)
+            file_name = base_file[0]
+            extension = base_file[1]
+            if file_name[0] != "." and extension == ".json":
+                extracted_json = json.loads(extracted_member.read().decode("utf-8"))
+                seed = extracted_json["seed"]
+
+                for timepoint in extracted_json["timepoints"]:
+                    one_timepoint = parse_growth_timepoint(timepoint, seed)
+                    for data in one_timepoint:
+                        all_timepoints.append(data)
+
+    timepoints_df = pd.DataFrame(all_timepoints, columns=GROWTH_COLUMNS)
+    return timepoints_df
+
+
+def convert_state_to_string(state_index: int) -> Union[str, None]:
```

Comment:
This can just be a list, since you have the indices. It would also make sense to be able to pass that list as a parameter, since other versions of ARCADE (e.g. CARCADE) have a slightly different mapping of cell state index to name.